web-ui/src/main/resources/catalog/templates/admin/harvest/harvest-settings.html

```diff
@@ -160,165 +173,237 @@
+                    <p data-translate="">
+                      metadataRecords
+                    </p>
+                  </li>
+                </ul>
 
-                  <pre><span data-ng-repeat="line in h.info[0]['stack']">{{line['@class']}} {{line['@file']}} ({{line['@line']}}) #{{line['@method']}}
-</span></pre>
-                </div>
+                <!-- TODO: loop over configurable list -->
+                <span data-ng-repeat="r in searchResults.facets"
+                      data-ng-if="r.key === 'availableInServices'">
+                  <ul class="col-md-4 text-center list-group gn-nomargin-bottom"
+                      data-ng-repeat="i in r.items">
+                    <li class="list-group-item">
+                      <h1>
+                        <i class="fa fa-download" data-ng-if="i.value === 'availableInDownloadService'"></i>
+                        <i class="fa fa-eye" data-ng-if="i.value === 'availableInViewService'"></i>
+                        {{ i.count }}</h1>
+                      <p>
+                        {{ 'availableInServices-' + i.value | translate }}
+                      </p>
+                    </li>
+                  </ul>
+                </span>
               </div>
 
-              <div class="timeline-heading" data-ng-if="h.info[0]['stack'] === undefined">
-                <h4 data-ng-show="h.deleted == 'true'">
-                  <strong>{{h.info[0][0]['@recordsRemoved']}}</strong>
-                  <span data-ng-show="h.info[0][0]['@recordsRemoved']" data-translate="">recordsRemoved</span></h4>
-                  <strong>{{h.info[0][0]['@recordsTransfered']}}</strong>
-                  <span data-ng-show="h.info[0][0]['@recordsTransfered']" data-translate="">recordsTransfered</span></h4>
-                <h4 data-ng-show="h.deleted == 'false'" class="timeline-title">
-                  <strong>{{h.info[0].total}}</strong>
-                  <span data-translate="">recordsHarvestedIn</span>
-                  <strong>{{h.elapsedtime}}</strong>
-                  <span data-translate="">seconds</span>
-                </h4>
-                <p>
-                  <small class="text-muted">
-                    <i class="fa fa-clock-o"></i>
-                    {{ h.harvestdate | gnFromNow }}
-                  </small>
-                </p>
-              </div>
-              <div class="timeline-body">
-                <p>
-                <ul>
-                  <li data-ng-repeat="(k, v) in h.info[0]"
-                      data-ng-show="v > 0">
-                    {{k}}: {{v}}
+              <div class="row">
+
+                <ul class=" list-group gn-nomargin-bottom height-auto">
+
+                  <li class="col-lg-6 col-md-12 list-group-item gn-margin-top"
+                      data-ng-repeat="f in searchResults.facets"
+                      data-ng-if="f.key === 'cl_hierarchyLevel.key' || f.key === 'tag.default'">
```

Comment:
Here you assume that those facet are in the config. It may not be true. We can start with this, but would be better to create a dedicated config to the harvester page search at some point.

Reply:
Yes, that was the plan, to have a config for this in the near future